### PR TITLE
Fix lint invocation in Jenkins

### PR DESCRIPTION
**Pull request by akalin-keybase**
Fix lint errors that slipped through.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -369,7 +369,11 @@ def testGo(prefix) {
             goversion = sh(returnStdout: true, script: "go version").trim()
             // Ideally, we'd do this on Windows, too, but we'd have to figure
             // out how to do it with batch files or PowerShell.
-            sh "make lint"
+            sh '''
```
Comment (akalin-keybase) [inline Jenkinsfile]: Copied from similar block in kbfs Jenkinsfile

File: go/profiling/timebuckets.go [hunk 2]
```diff
@@ -48,7 +48,7 @@ func (t *TimeBuckets) Log(ctx context.Context, bucketName string) {
 	t.log.CDebugf(ctx, "%s [time=%s]", bucketName, t.Get(bucketName))
 }
 
-type finFn func()
+type FinFn func()
```
Comment (mlsteele) [inline go/profiling/timebuckets.go]: What happened here?

Comment (mlsteele) [reply go/profiling/timebuckets.go]: Nevermind

Comment (akalin-keybase) [reply go/profiling/timebuckets.go]: Lint complained about returning unexported types

**Pull request merged**